src/distributions/mod.rs

```diff
@@ -189,9 +189,29 @@ impl<'a, T, D: Distribution<T>> Distribution<T> for &'a D {
 /// let mut rng = thread_rng();
 /// let mut erased_rng: &mut RngCore = &mut rng;
 /// let val: f32 = erased_rng.sample(Uniform);
-/// println!("f32 from [0,1): {}", val);
+/// println!("f32 from (0,1): {}", val);
 /// ```
 ///
+/// # Open interval for floats
+/// In theory it is possible to choose between an open interval `(0, 1)`, and
+/// the half-open intervals `[0, 1)` and `(0, 1]`. All can give a distribution
+/// with perfectly uniform intervals. Many libraries in other programming
+/// languages default to the closed-open interval `[0, 1)`. We choose here to go
+/// with *open*, with the arguments:
+///
+/// - The chance to generate a specific value, like exactly 0.0, is *tiny*. No
+///   (or almost no) sensible code relies on an exact floating-point value to be
+///   generated with a tiny chance (1 in 2^23 for `f32`, and 1 in 2^52 for
+///   `f64`). What is relied on, is on the distribution to be uniform, and have
```

Comment:
What is relied on is having a uniform distribution and a mean of 0.5.

Why median??

Reply:
I was shuffling the names... But you are right, median is not the one.

Reply:
Changed b.t.w., but the line is just below your comment.

Reply:
Semantically correct but poor grammar, hence my suggestion, but never mind, this'll do.

Reply:
A, now I get your comment!
This grammar is perfectly fine in Dutch though...